Dockerfile

```diff
@@ -1,13 +1,21 @@
-FROM node:9.2.0-alpine
+FROM node:9-slim
 
-ENV SCREENIE_VERSION=1.2.0
+ENV SCREENIE_VERSION=2.0.0
+ENV SCREENIE_CHROMIUM_ARGS="--no-sandbox"
 
-RUN apk add --no-cache fontconfig curl ca-certificates \
-    && curl -Ls "https://github.com/dustinblackman/phantomized/releases/download/2.1.1/dockerized-phantomjs.tar.gz" | tar xz -C / \
-    && npm install -g screenie-server@$SCREENIE_VERSION --unsafe-perm \
-    && apk del curl \
-    && rm -rf /tmp/*
+RUN apt-get update && \
+apt-get install -yq gconf-service libasound2 libatk1.0-0 libc6 libcairo2 libcups2 libdbus-1-3 \
+libexpat1 libfontconfig1 libgcc1 libgconf-2-4 libgdk-pixbuf2.0-0 libglib2.0-0 libgtk-3-0 libnspr4 \
+libpango-1.0-0 libpangocairo-1.0-0 libstdc++6 libx11-6 libx11-xcb1 libxcb1 libxcomposite1 \
+libxcursor1 libxdamage1 libxext6 libxfixes3 libxi6 libxrandr2 libxrender1 libxss1 libxtst6 \
+ca-certificates fonts-liberation libappindicator1 libnss3 lsb-release xdg-utils wget
+
+RUN wget https://github.com/Yelp/dumb-init/releases/download/v1.2.0/dumb-init_1.2.0_amd64.deb
```

Comment:
Add the two RUN commands into the single one above, btw. Makes the image a bit more lightweight.